www/front_src/src/App/index.tsx

```diff
@@ -34,13 +34,14 @@ const useStyles = makeStyles((theme) => ({
     zIndex: 1500,
   },
   mainContent: {
-    backgroundColor: theme.palette.background.paper,
-    height: `calc(100vh - ${theme.spacing(headerHeight + headerPadding)})`,
+    height: `calc(100vh - ${theme.spacing(headerHeight)})`,
     width: '100%',
   },
   wrapper: {
     alignItems: 'stretch',
+    backgroundColor: theme.palette.background.default,
```

Comment:
```suggestion
    backgroundColor: theme.palette.background.paper,
```

Reply:
I don't understand : white is the default background color for light theme, paper background (#f4f4f4) is used for panels, submenus, etc.